spec/features/events/visitor_can_search_for_events_by_zipcode_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe do
+  context "visitor" do
+    it "views events for a given zipcode" do
+      VCR.use_cassette("events") do
+        Net::HTTP.get_response(URI("http://api.eventful.com/json/events/search?&location=80202&sort_order=popularity&app_key=#{ENV["EVENTFUL_API_KEY"]}&date=march&sort_direction=descending&page_size=16&include=popularity"))
```

Comment:
Everything in this test looks great... checking for the count of the CSS class, scoping to a section of the page, using VCR etc. I'm unclear why there's an API call happening here. I'm guessing it's a relic from some experiments.